string axis_name
float32 change_value
float32 duration
---
bool success
string message